### PR TITLE
Fix Links for Developer Section

### PR DESCRIPTION
**Pull request by Abhisek-**
Fixed all the Developer section links in following files
* Help/FAQ.rst
* Users/Install.rst
* Users/Tutorial.rst

Fixes https://github.com/coala/documentation/issues/349

### CODE REVIEW

**Comment by Mariatta** [comment]
Looks good to me! Thanks @Abhisek-

**Comment by Mariatta** [comment]
ack d454511

**Comment by Mariatta** [comment]
unack d454511

File: Help/FAQ.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ that will teach you everything you need to know to use coala.
 If you're willing to contribute and become a part of our coalaian community,
 we have written a guide that will help you fix an issue on your own, just by
 following and understanding the indications
-:doc:`here <../Developers/Newcomers_Guide>`.
+`here <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`_.
```
Comment (Mariatta) [inline Help/FAQ.rst]: This should be coala.io/newcomer

File: Users/Install.rst [hunk 2]
```diff
@@ -119,7 +119,7 @@ Installing coala from source
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you would like to develop coala, you should check out our
-:doc:`Newcomer Tutorial <../Developers/Newcomers_Guide>` and
+`Newcomer Tutorial <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`_ and
```
Comment (Mariatta) [inline Users/Install.rst]: http://coala.io/newcomer

**Comment by Abhisek-** [comment]
Is there anything else that I need to do @Mariatta

File: Users/Tutorial.rst [hunk 3]
```diff
@@ -380,7 +380,7 @@ If you need more flexibility, know that coala is extensible in many ways
 due to its modular design:
 
 -  If you want to write your own bears, take a look at
-   :doc:`our tutorial <../Developers/Writing_Bears>`.
+   `our tutorial <http://api.coala.io/en/latest/Developers/Writing_Bears.html>`_.
```
Comment (Mariatta) [inline Users/Tutorial.rst]: http://coala.io/writingbears

**Comment by Abhisek-** [comment]
That redirects to the same page. Is there any specific reason to change to that address. Just curious. Anyway will do it

**Comment by Mariatta** [comment]
The short url is preferred so that if we change the location of the documentation, we can just change the url redirection, and not have to go through all of the documentation text again. Hope this makes sense :)
Thanks.

**Comment by Abhisek-** [comment]
thanks @Mariatta that makes sense. I've changed the requested changes

**Comment by nemaniarjun** [comment]
ack 2031e4d7ef1038dfdd689d8a74f94724abdab61a

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**